README.md

```diff
@@ -1,183 +1,218 @@
+[![Build Status](https://travis-ci.org/larriereguichet/AdminBundle.svg?branch=master)](https://travis-ci.org/larriereguichet/AdminBundle)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/?branch=master)
+[![SensioLabsInsight](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0/mini.png)](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0)
+
+
 # AdminBundle
 
-AdminBundle allows you to create flexible and robust management application, using a simple configuration,
+AdminBundle allows you to create flexible and robust backoffice application, using a simple yml configuration,
```

Comment:
The AdminBundle allows to create a flexible and robust backoffice on any Symfony application, with simple `yml` configuration only.